httpx/models.py

```diff
@@ -646,11 +644,7 @@ def __repr__(self) -> str:
         """
         Read and return the request content.
         """
-        return await self.content.aread()
-
-    async def stream(self) -> typing.AsyncIterator[bytes]:
-        async for part in self.content:
-            yield part
+        return await self.stream.aread()
```

Comment:
Not a direct consequence of this PR, but should this method on the request really be called `.aread()`? As in: will we need a sync `def read(self):` for sync support?

Reply:
It should yes. It's also not *actually* clear if we should expose this method at all.

Reply:
Well, I'm not sure we actually. Do we use it anywhere internally (in the package or in the tests) already?